### DISCUSSION TITLE
Airflow ansible project sharing

### DISCUSSION THREAD

**Discussion by ginolegigot** [apache/airflow#35180] [Q&A]
Hello,
I'd like to share an ansible airflow celery cluster project.  
Is there a particular place to do this where some people from apache airflow could review or i just drop the code in my personal repo ?

**Comment by Taragolis** [comment]
You could add it in [Airflow Ecosystem](https://airflow.apache.org/ecosystem/#other-deployments-methods), just by simple click on `Suggest a change on this page` and make PR changes into the Github Editor
